### PR TITLE
Manager convenience property

### PR DESCRIPTION
**Pull request by iamanikeev**

### CODE REVIEW

**Comment by iamanikeev** [comment]
This helps to avoid unnecessary runs of `get_by_id()` where we don't need and entire instance, but want to call some method knowing the id.

File: djoffers/models.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ class HasOffersModel(models.Model):
     """
     Basic entity to store on our side. It points to remote HasOffers entity via integer ID.
     """
-    hasoffers_id = models.IntegerField(unique=True)
+    hasoffers_id = models.IntegerField(unique=True, null=True)
```
Comment (Stranger6667) [inline djoffers/models.py]: Is it possible to have an object in HO without an ID? For me it doesn't make sense

**Comment by Stranger6667** [comment]
Could you, please, provide an example?

File: djoffers/models.py (hunk 1 above)
Comment (iamanikeev) [reply djoffers/models.py]: This diff shouldn't be here, removed. It is possible though. When you have a model that is being synced from local DB to HO.

File: djoffers/models.py [hunk 2]
```diff
@@ -35,4 +35,10 @@ def instance(self):
         """
         Convenience wrapper for remote instance. Taken from `pyoffers`.
         """
-        return self.hasoffers._managers[self.manager_name].find_by_id(self.hasoffers_id)
+        if self.hasoffers_id is None:
```
Comment (Stranger6667) [inline djoffers/models.py]: It can't be `None`

**Pull request merged**